tools/afl-fuzzer/inputs/input1.txt

```diff
@@ -0,0 +1 @@
+100 4 aaaa
```

Comment:
What is this file for ?

Reply:
I have removed this file in the latest commit [d066e0b](https://github.com/thesofproject/sof/pull/3100/commits/d066e0b589ab36d2da690967684cb3056cce9c6d).